src/frontend/screens/SignInScreen/SignInScreen.js

```diff
@@ -1,8 +1,71 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
Separate the local imports with module imports will increase the readability

Example:
import { Link, useNavigate } from "react-router-dom";

import "./SignInScreen.css";
import { useAuth } from "../../contexts";

Reply:
yup would take care of it